docs/contribute/documentation.md

```diff
@@ -40,11 +58,6 @@ Intellij content here
 {% endcapture %}
 {% include ide-tabs.html eclipse=eclipse intellij=intellij %}
 
-
-**Attention:** The major disadvantage of this solution is that we use [jekyll liquid tags](https://jekyllrb.com/docs/liquid/).
```

Comment:
Is this no longer true? Or have you just removed the hint as we now also have a doc build to check this? Also, does the doc build job still work with this new setup? And does it still work the same or does the new deployment logic lead to issues (like the doc test build deploying the page)?

Reply:
> Or have you just removed the hint as we now also have a doc build to check this?

Yes, because we check this

> Also, does the doc build job still work with this new setup?

The results on my fork are created with the job. It should work.

> And does it still work the same or does the new deployment logic lead to issues

The action still uses the same triggers (all branches but restricted to changes in the `docs` directory) and all deploy related parts use the condition `if: github.ref == 'refs/heads/master' `.
GitHub Actions skips the step if the condition is not met (as [here](https://github.com/m273d15/saros/runs/573861915?check_suite_focus=true)).

However, the build logic is changed. Instead of using a docker image we install the jekyll dependencies. I tried multiple alternatives (e.g. with the `jekyll/jekyll` image that is cached by default), but it seems like just installing the dependencies is faster. Furthermore,
the used approach has the advantage that dependency versions are the same in CI and the local development environment (as the jekyll version).